lucene/CHANGES.txt

```diff
@@ -648,6 +648,8 @@ Optimizations
 
 Bug Fixes
 ---------------------
+* LUCENE-10008: Respect ignoreCase in CommonGramsFilterFactory (Vigya Sharma)
```

Comment:
Whoa, this is in the wrong (Lucene 8.6.0) section of CHANGES!

Since the 8.10 bits have already been set free, can you move this to the new 8.11 section?